packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -46,4 +46,179 @@ export const components: ThemeOptions["components"] = {
       },
     },
   },
+  MuiButton: {
+    defaultProps: {
+      disableElevation: true,
+      disableRipple: true,
+    },
+    variants: [
+      {
+        props: { variant: "primary" },
+        style: {
+          fontWeight: 600,
+          color: "#fff",
```

Comment:
Could you update these shortcodes to the full `#ffffff`? That'll make it easier for us to mass find/replace later on.